modules/shared/facts.d/win_env_fact.ps1

```diff
@@ -0,0 +1,27 @@
+# Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I would argue that this "shared" module is very windows specific since it has powershell facts.  I would rename it win_shared or win_common

Reply:
Moved to win_shared.